packages/components/.storybook/manager.js

```diff
@@ -0,0 +1,5 @@
+import { addons } from '@storybook/addons';
+import setup from '../../../presets/react/storybook/manager';
+import pkg from "../package.json";
+
+setup({ pkg, addons });
```

Comment:
addons holds some kind of store, if I import it in presets/react/storybook/manager, the store will be different to the one used in this package